discover_overlay/locales/fr/LC_MESSAGES/default.po

```diff
@@ -424,7 +424,7 @@ msgid "Integrate with Pipewire/Pulseaudio"
 msgstr "Intégration avec Pipewire/Pulseaudio"
 
 msgid "Core"
-msgstr "Noyau"
+msgstr "Réglages"
```

Comment:
Again, derives from the source string.

Reply:
I'd like this to convey the idea that the settings here are applied equally to each different section prior. I'm not certain the English does this either, right now.

I'm also considering making this the default opened tab and having the About/Overview become the last in the list

Reply:
Good idea, I’m not sure what to call it either so I’ll leave you to change that.